stage1/init/pod.go

```diff
@@ -465,6 +465,8 @@ func (p *Pod) appToNspawnArgs(ra *schema.RuntimeApp) ([]string, error) {
 	id := ra.Image.ID
 	app := ra.App
 
+	var err error
```

Comment:
This seems like unnecessary scope leak, can you just declare it (if at all) on L519?